dbms/src/Storages/DeltaMerge/SkippableBlockInputStream.cpp

```diff
@@ -45,6 +45,14 @@ ConcatSkippableBlockInputStream<need_row_id>::ConcatSkippableBlockInputStream(
     current_stream = children.begin();
 }
 
+template <bool need_row_id>
+void ConcatSkippableBlockInputStream<need_row_id>::appendChild(SkippableBlockInputStreamPtr input, size_t rows)
+{
+    children.push_back(input);
+    this->rows.push_back(rows);
```

Comment:
Why not just change the name of the parameter?

Reply:
Fixed